shared/protos/Path.proto

```diff
@@ -12,7 +12,7 @@ message Path {
 
   optional int32 id = 2 [(gmx) = "GMX_DEPRECATED"];
 
-  optional string background_room_name = 3 [(resource_ref)="room", (gmx_id) = "backroom"];
+  optional string background_room_name = 3 [(resource_ref)="room", (gmx) = "backroom"];
```

Comment:
The `gmx_id` was unimplemented because it wasn't necessary in the command line. I believe the intention was for `_id` suffix to mean convert the string to an id.

Reply:
well this is the only field like this in gmx so having an attribute for it doesn't seem necessary

Reply:
It still looks cleaner, and because of options/texture pages/audio groups that might not be the case. I have not truly seen 100% of the GMX format.

Regardless, this was in fact a mistake by YoYoGames, I am sure. If you create and delete a bunch of rooms, then set the path background room, save and reopen, it can be a different room after reopening. Since GMX doesn't save the ids, they are defragged when the project is reopened.

Nothing we can do about it now though.

Reply:
right its just a workaround for bug yoyo has in gmx. Every other resource we've seen in gmx uses a string so until another one appears im not interested in using an attribute

Reply:
Well we could have lgm save the room as a string / another attribute to ensure at-least lgm doesn't  de-synced the room name/id but I don't think it's super important

Reply:
Sure, would be interesting to see if GMSv1.4 could read it too.